packages/miniflare/src/runtime/config/generated.ts

```diff
@@ -4,7 +4,8 @@ import * as $ from "capnp-es";
 export const _capnpFileId = BigInt("0xe6afd26682091c01");
 /**
  * Top-level configuration for a workerd instance.
- * */
+ *
```

Comment:
were the changes in this file all generated somehow? Seems to be mostly comments

Reply:
Yes, this code is generated by `capnp-es` from `workerd.capnp`.
There are minor formatting changes in the new v0.0.11.
The few actual changes are related to capnp file updates.